src/label.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (C) 2023, TomTom (http://tomtom.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+type TLabelCategory = "bump" | "type" | "scope" | "initial development";
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export abstract class Label {
```

Comment:
I'm confused; this really has no business being a class, yet you made it one and specifically disabled the (justified) typescript warning?
I'd just go for `getLabelCategory`, `isLabelVisible`, etc.
Or, if you really want to put the functions in an object, you can remove the class, `s/static/export function/g`, and import this module like:
```
import * as githubLabel from './label';
```